### PR TITLE
Market creation deadlines input + liquidity page using sdkv2

### LINKED ISSUE #330: Integrate grace period, oracle duration, and dispute duration
Upcoming in the next release, the dispute period will now be configurable by the market creator. The new configurable variables are `grace_period`, `oracle_duration`, and `dispute_duration`. 

See https://github.com/zeitgeistpm/zeitgeist/pull/732 for context.

## Needed Changes

- [ ] Add the new configuration options to the market creation flow.
- [ ] Display the configured values on the market page with information about what each of the configuration values will mean for how the market is resolved.

### PR DESCRIPTION
**Pull request by yornaath**
**Work done:**
- [x] Implement input UI for deadlines.
- [x] Implement inputs in market creation page/form.
- [x] Convert dates and durations to block durations.
- [x] Use custom date when signing market creation extrinsic.
- [x] Load liquidity pools using sdkv2
- [x] Implement newer liquidity pools list design.
- [x] Load total liquidity after fetching first page in the list so we dont rely on the paginated items for calculating.
- [x] Bring back pagination.

**_Note: liquidity pools with sdkv2 branch merged into this. Relevant files for this pr is `pages/create.tsx` and `components/create/MarketDeadlinesInput.tsx`_**

Fixes #330
Fixes #374

### CODE REVIEW

**Comment by yornaath** [comment]
Blocked 🛑  Waiting for `market.deadlines` to be available on mainnet indexer. Let me know @saboonikhil

**Comment by saboonikhil** [comment]
> Blocked 🛑 Waiting for `market.deadlines` to be available on mainnet indexer. Let me know @saboonikhil 

🟢

File: components/create/MarketDeadlinesInput.tsx [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+};
+
+export type GracePeriodValue =
+  | {
+      label: "None";
+      value: 0;
+    }
+  | {
+      label: "1 Hour";
+      value: 300;
```
Comment (TvrtkoM) [inline components/create/MarketDeadlinesInput.tsx:28]: how is this value calculated? in which units is it? 1 hour = 3600 seconds.

File: components/create/MarketDeadlinesInput.tsx [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    }
+  | {
+      label: "1 Day";
+      value: 7200;
```
Comment (TvrtkoM) [inline components/create/MarketDeadlinesInput.tsx:32]: 1 day = 86400 seconds.

File: components/create/MarketDeadlinesInput.tsx [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+    };
+
+export type OracleAndDisputePeriodValue =
+  | {
+      label: "1 Day";
+      value: 7200;
+    }
+  | {
+      label: "4 Days";
+      value: 28800;
```
Comment (TvrtkoM) [inline components/create/MarketDeadlinesInput.tsx:46]: this seems to be 8 hours, not 4 days, 4 days = 345600 seconds

**Review by TvrtkoM — COMMENTED** [review]
Can you please double check how the values are set. I just created a market on local environment and for:
```
grace - 1 day
oracle - 5 days, 4 hours
dispute duration - 8 days, 20 hours
```
indexer is showing
```
        "deadlines": {
          "disputeDuration": "63600",
          "gracePeriod": "7200",
          "oracleDuration": "37200"
        },
```

File: components/create/MarketDeadlinesInput.tsx [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+          <div className="bg-sky-200 dark:bg-black text-ztg-14-150  -ml-2 font-bold rounded-ztg-5 h-ztg-40 p-ztg-8 font-lato focus:outline-none border-1 dark:border-black text-black dark:text-white text-right  w-14">
+            DAYS
+          </div>
+        </div>
+        <div
+          className={`flex w-32 mr-4 border-1 ${
+            props.value.label === "Custom" && "border-gray-700 rounded-ztg-5"
+          } ${!props.isValid && "border-1 !border-vermilion"}`}
+        >
+          <input
```
[the remark this answers is not in the document]
Comment (yornaath) [reply components/create/MarketDeadlinesInput.tsx:318]: Ah yeah for sure

**Comment by yornaath** [comment]
> Can you please double check how the values are set. I just created a market on local environment and for:
> 
> ```
> grace - 1 day
> oracle - 5 days, 4 hours
> dispute duration - 8 days, 20 hours
> ```
> 
> indexer is showing
> 
> ```
>         "deadlines": {
>           "disputeDuration": "63600",
>           "gracePeriod": "7200",
>           "oracleDuration": "37200"
>         },
> ```
> 
> Edit: this actually looks fine if 1 hour is considered to be 300 units as blocks. Didn't know this until now. :)

Yeah I think this looks correct

**Pull request merged**